swim/memberlist.go

```diff
@@ -352,10 +406,16 @@ func (m *memberlist) getJoinPosition() int {
 }
 
 // applies a change directly to the member list
-func (m *memberlist) Apply(change Change) {
+func (m *memberlist) Apply(change Change) bool {
```

Comment:
Oh I was a bit confused on how this function works. I thought apply checked if the change has precedence over the current member. Maybe we can add a comment about that too though